src/app/components/Container.tsx

```diff
@@ -2,10 +2,11 @@ type MaxWidth = "sm" | "md" | "lg" | "xl" | "2xl";
 
 type Props = {
   children: React.ReactNode;
+  screen?: boolean;
```

Comment:
Can we call it `isScreenView` to indicate it's boolean and it's related to views? Otherwise it's too generic I'm worried.

Reply:
Done!